types/unist/index.d.ts

```diff
@@ -6,13 +6,14 @@
 //                 Titus Wormer <https://github.com/wooorm>
 //                 Junyoung Choi <https://github.com/rokt33r>
 //                 Ben Moon <https://github.com/GuiltyDolphin>
+//                 JounQin <https://github.com/JounQin>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.0
 
 /**
  * Syntactic units in unist syntax trees are called nodes.
  */
-export interface Node {
+export interface Node<NData extends object = Data> {
```

Comment:
Please document the new type parameters in each case. E.g., see https://github.com/unifiedjs/unified/blob/350ed9dfe109c9c12f4779f3ee89d63cd66843d0/types/index.d.ts#L247.

Reply:
Done